src/wrapper.cpp

```diff
@@ -1,40 +1,52 @@
 #include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <pybind11/operators.h>
+
+#include "Pet.h"
+#include "Ctxt.h"
 
 int add(int i, int j) {
     return i + j;
 }
 
 namespace py = pybind11;
 
+using namespace pybind11::literals;
+using namespace std;
+
 PYBIND11_MODULE(hepy, m) {
+
+    // Wrapping for class SKHanlde in Ctxt.h
+    py::class_<SKHandle>(m, "SKHandle")
+      .def(py::init<long, long, long>())
+      .def("setBase", &SKHandle::setBase)
+      .def("isBase", &SKHandle::isBase)
+      .def("setOne", &SKHandle::setOne)
+      .def(py::self == py::self)
+      .def(py::self != py::self)
```

Comment:
wut?

Reply:
Hah yes. This is the pybind11 magic setup for porting a C++ operator overload. `SKHandle::operator==` is ported into Python with `.def(py::self == py::self)`

Reply:
ic. so those 2 are wrappers for `==` operator and `!=` operator overloads